### DISCUSSION TITLE
Change default height of Mantine Tiptap-based RichTextEditor Content box

### DISCUSSION THREAD

**Discussion by gurbaaz27** [mantinedev/mantine#4319] [Q&A] [answered]
I'm using Tiptap based rich text editor provided by Mantine [here](https://mantine.dev/others/tiptap/), and I am not able to figure out how to increase the default height of the Content box. Sure, as the text content grows the box height increases with it, but I want to keep it a certain height even when there is no or little content in the editor box. As you can see in the image below, the height is very short (only one row), but I want to keep it around 5-6 rows at the start itself.

![image](https://github.com/mantinedev/mantine/assets/60743677/645ea6e1-e12d-43b2-950f-a8255b3ba6a7)

**Comment by rtivital** [comment, marked as answer]
You can customize component styles with Styles API. Try setting min-height property.

**Comment by gurbaaz27** [comment]
@rtivital thanks, I set the min-height of content box with `styles={{ content: { minHeight: '12rem' } }}`, and it seems to work. 

Apart from that, I have one other query. As the content increases, the height of content-box is increasing and it overflows from screen at certain point. I want a fixed max height (I think that is achievable with max-height property), but I also want to be able to scroll inside the box of the rich text editor, so that I am able to navigate to across contents. I dont think the content box is scrollable by default. Any help here would be appreciated!
